Jenkinsfile

```diff
@@ -78,7 +78,7 @@ def stablePresubmit(gitUtils, bazel, utils) {
     sh('ln -s ~/.kube/config platform/kube/')
     stage('Integration Tests') {
       timeout(30) {
-        sh("bin/e2e.sh -count 10 -debug -tag debug${gitUtils.GIT_SHA} -v 2")
```

Comment:
we are rebuilding them at https://github.com/istio/manager/blob/master/Jenkinsfile#L95